### ISSUE TITLE
Mummy death curse affects wrong slot

### ISSUE THREAD

**Issue by Sandman25DCSS** [jeremygurr/dcssca#331]
I got `Your ring 7 slot has been cursed.` but I am not an octopode, I am DD. I do wear macabre finger necklace if it matters

**Comment by Sandman25DCSS** [comment]
And I see that no my ring slots are cursed.

**Comment by jeremygurr** [comment]
maybe ring 7 slot goes with the finger necklace? Does the finger necklace give you another ring slot?

**Comment by yrmvgh** [comment]
Yeah, I've had slots that don't exist get cursed every now and then on multiple characters. I forgot to post it as an issue.

**Comment by yrmvgh** [comment]
You see it often in that timed low-level hissing sand portal, for obvious reasons

**Comment by Sandman25DCSS** [comment]
None of my rings were cursed after the message. Also I got similar messages about ring slot 6.

**Comment by jeremygurr** [comment]
finally fixed
